src/pudl/transform/phmsagas.py

```diff
@@ -0,0 +1,164 @@
+        "report_year",
+        "report_number",
+        "operator_id_phmsa",
+        "excavation_damage_excavation_practices",
+        "excavation_damage_locating_practices",
+        "excavation_damage_one_call_notification",
+        "excavation_damage_other",
+        "excavation_damage_total",
+        "excavation_tickets",
+        "services_efv_in_system",
+        "services_efv_installed",
+        "services_shutoff_valve_in_system",
+        "services_shutoff_valve_installed",
+    ],
+    "columns_with_nas_to_fill": [
+        "excavation_damage_excavation_practices",
+        "excavation_damage_locating_practices",
+        "excavation_damage_one_call_notification",
+        "excavation_damage_other",
+        "excavation_damage_total",
+        "excavation_tickets",
+        "services_efv_in_system",
+        "services_efv_installed",
+        "services_shutoff_valve_in_system",
+        "services_shutoff_valve_installed",
+        "federal_land_leaks_repaired_or_scheduled",
+    ],
+    "capitalization_exclusion": ["headquarters_address_state", "office_address_state"],
+    "cols_for_state_standardization": [
+        "headquarters_address_state",
+        "office_address_state",
+    ],
+}
+
+##############################################################################
+# PHMSAGAS transformation logic
+##############################################################################
+
+
+@asset(
+    ins={"raw_data": AssetIn(key="raw_phmsagas__yearly_distribution")},
+    io_manager_key=None,  # TODO: check on this one ("pudl_io_manager"? something else?)
+    compute_kind=None,  # TODO: check on this one
+)
+def core_phmsagas__yearly_distribution_operators(
+    raw_data: pd.DataFrame,
+) -> pd.DataFrame:
+    """Build the :ref:`core_phmsagas__yearly_distribution_operators`."""
+    df = raw_data[YEARLY_DISTRIBUTION_OPERATORS_COLUMNS["columns_to_keep"]]
+    df[YEARLY_DISTRIBUTION_OPERATORS_COLUMNS["columns_to_convert_to_ints"]] = df[
+        YEARLY_DISTRIBUTION_OPERATORS_COLUMNS["columns_to_convert_to_ints"]
+    ].astype("Int64")
+
+    # Ensure all "report_year" values have four digits
+    mask = df["report_year"] < 100
+
+    # Convert 2-digit years to appropriate 4-digit format (assume cutoff at year 50)
+    # We could also use the first 4 digits of the "report_number" but there was at least one anomaly here with an invalid year
+    df.loc[mask, "report_year"] = df.loc[mask, "report_year"].apply(
+        lambda x: 2000 + x if x < 50 else 1900 + x
+    )
+
+    # Fill NA values with zeroes because these columns are simply counts.
+    # Note that "excavation_damage..." columns should sum up to the value in "excavation_damage_total". However, many rows
+    # (on the scale of thousands) do not actually sum up to "excavation_damage_total".
+    df[YEARLY_DISTRIBUTION_OPERATORS_COLUMNS["columns_with_nas_to_fill"]] = df[
+        YEARLY_DISTRIBUTION_OPERATORS_COLUMNS["columns_with_nas_to_fill"]
+    ].fillna(0)
```

Comment:
We distinguish between NA as "not reported" and 0 as "reported as 0", so filling this would actually change the meaning here (from "I didn't fill this in" to "I had 0 incidents"). Please drop this step.

Reply:
Updated